README.md

```diff
@@ -108,13 +109,63 @@ If you’re starting totally new, we got you! Here’s a 101 conceptual read on
 
 Find nice articles with rich and valuable content about different topics related to Filecoin network.
 
-## Looking for a JS library to interact with Filecoin? :wrench::hammer:
+## Looking for the complete documentation? :books::books:
+
+Zondax documentation: [Let's go to docs web](https://docs.zondax.ch/fevm/filecoin-solidity) :arrow_upper_right:
```

Comment:
we should be driving the users toward the filecoin docs: https://docs.filecoin.io/smart-contracts/developing-contracts/solidity-libraries/ since the Zondax ones aren't supported anymore and we have no control over it, if we need to do any edits on it.